app/views/courses/preview/_train_with_disabilities.html.erb

```diff
@@ -4,7 +4,7 @@
     <% if @provider.train_with_disability.present? %>
       <%= markdown(@provider.train_with_disability) %>
     <% else %>
-      <p class="app-missing-section">Please add details about <%= link_to 'training with a disability', manage_ui_url("/organisation/#{@provider.provider_code}/about") %>.</p>
+      <p class="app-missing-section">Please add details about <%= link_to 'training with a disability', about_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year) %>.</p>
```

Comment:
```suggestion
      <p class="app-missing-section">Please add details <%= link_to 'about your organisation', about_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year) %>.</p>
```

based on https://github.com/DFE-Digital/search-and-compare-ui/blob/052df97b158ad7afa32ee15bd855d7482a0d815d/shared/Views/Shared/Components/CourseDetails/_TrainWithDisabilities.cshtml#L23

@fofr

Reply:
What's there already is correct, ie this PR is correct.